MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/ig/SnapshotGridActivity.java

```diff
@@ -0,0 +1,142 @@
+package com.mapbox.mapboxandroiddemo.examples.ig;
```

Comment:
I believe the SnapshotGridActivity and its layout are unrelated to this PR and could be removed

Reply:
yea, we got rid of that example anyways. Not sure why/how it stuck around on this pr. Removed